feat(contactus): add smooth scrolling

dcshzj opened the pull request:
## Problem

<!-- What problem are you trying to solve? What issue does this close? -->

Closes IS-544.

## Solution

<!-- How did you solve the problem? -->

**Breaking Changes**

<!-- Does this PR contain any backward incompatible changes? If so, what are they and should there be special considerations for release? -->

- [ ] Yes - this PR contains breaking changes
- [x] No - this PR is backwards compatible with ALL of the following feature flags in this [doc](https://www.notion.so/opengov/Existing-feature-flags-518ad2cdc325420893a105e88c432be5)

**Improvements**:

- Smooth scrolling has been added to the contact us page.

## Before & After Screenshots

https://github.com/isomerpages/isomercms-frontend/assets/27919917/9044833b-13ad-41a7-b9b8-89e9c53558b1

## Tests

<!-- What tests should be run to confirm functionality? -->

- [ ] Unit tests (using `npm run tests`)
- [ ] e2e tests (comment on this PR with the text `!run e2e`)
- [x] Smoke tests
    - [ ] Navigate to the contact us page for any site
    - [ ] Make changes to existing blocks and observe for smooth scrolling to that block on the preview
    - [ ] Add a new block and observe for smooth scrolling to the new block

## Deploy Notes

<!-- Notes regarding deployment of the contained body of work.  -->
<!-- These should note any new dependencies, new scripts, etc. -->

*None*

alexanderleegs reviewed (changes requested):
Scrolling for expanding a component still throws an error!

dcshzj commented:
@alexanderleegs turns out it has been broken all along since the new stylings came in. Fixed and also removed the unneeded displaySections stuff now that we are using accordion.

alexanderleegs reviewed (approved):
lgtm!
